app.js

```diff
@@ -130,7 +130,8 @@ io.on('connection', (socket) => {
           studentsCopy = [...students].map((student, idx) => {
             const studentCopy = { ...student };
 
-            if (idx === numOfStudents - 1) {
+            // if (idx === numOfStudents - 1) {
```

Comment:
why comment out instead of removing line?

Reply:
Outer condition is confusing as to why its even needed, but breaks app when removed. Since the original intent is hard to understand, I left it commented out as a reference. Ideally, the whole socket event function should be reworked.